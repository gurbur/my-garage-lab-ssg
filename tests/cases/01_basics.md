# 가장 큰 제목 (H1)

This is first paragraph. 마크다운의 문단은 한 개 이상의 연속된 텍스트 줄로 구성됩니다.

이렇게 빈 줄을 사이에 두면, 새로운 문단이 시작됩니다.

## 중간 제목 (H2)

*이텔릭체*와 **볼드체**, 그리고 ***이텔릭과 볼드를 동시에*** 사용할 수 있습니다. `인라인 코드`도 처리해야 합니다.

---

### 작은 제목 (H3)

수평선 위아래로 다른 내용이 올 수 있습니다.
