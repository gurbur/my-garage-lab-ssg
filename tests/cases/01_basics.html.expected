<h1>가장 큰 제목 (H1)</h1>
<p>This is first paragraph. 마크다운의 문단은 한 개 이상의 연속된 텍스트 줄로 구성됩니다.</p>
<p>이렇게 빈 줄을 사이에 두면, 새로운 문단이 시작됩니다.</p>
<h2>중간 제목 (H2)</h2>
<p><em>이텔릭체</em>와 <strong>볼드체</strong>, 그리고 <em><strong>이텔릭과 볼드를 동시에</strong></em> 사용할 수 있습니다. <code>인라인 코드</code>도 처리해야 합니다.</p>
<hr>
<h3>작은 제목 (H3)</h3>
<p>수평선 위아래로 다른 내용이 올 수 있습니다. </p>
